libs/cardano-ledger-binary/src/Cardano/Ledger/Binary/Decoding/Decoder.hs

```diff
@@ -529,7 +529,7 @@ decodeRecordNamedT name getRecordSize decoder = do
   lenOrIndef <- lift decodeListLenOrIndef
   x <- decoder
   lift $ case lenOrIndef of
-    Just n -> matchSize (Text.pack "Record " <> name) n (getRecordSize x)
+    Just n -> matchSize (Text.pack "Record " <> name) (getRecordSize x) n
```

Comment:
Argument positions have been swapped to correct the earlier mistake.

Reply:
Good catch